tests/storage/storageserver_test.py

```diff
@@ -250,6 +250,33 @@ def test_glusterfs_cli_missing(self, monkeypatch):
         gluster = GlusterFSConnection(id="id", spec="192.168.122.1:/music")
         assert gluster.options == ""
 
+    @pytest.mark.parametrize("userMountOptions", [
+        '',
+        'backup-volfile-servers=192.168.122.1:192.168.122.2',
+    ])
+    def test_glusterfs_backup_serv_connection(self, monkeypatch,
+                                              userMountOptions):
+        monkeypatch.setattr(storageServer, 'supervdsm', FakeSupervdsm())
+        monkeypatch.setattr(gluster_cli, 'exists', lambda: True)
+
+        def glusterVolumeInfo(volname=None, volfileServer=None):
+            assert volname == "music"
+            if volfileServer is not None:
+                raise ge.GlusterException()
```

Comment:
This is nice way to fail the test on the first try, but it is hidden in the middle of the test, so it is hard to understand the purpose of the test.

I think we should rename the test and add docstring like other tests explaning the purpose of the test.

```python
def test_glusterfs_retry_withotut_volfile_server(...)
    """
    The test will fail if called with volfileserver, simulating the case when
    a remote server is down. If the caller try again without volfileserver the
    call will succeed, simulating the case when running on a hyperconverged
    system when the local glsuter glsuter client can use one of the connected
    servers.
    """
```

Reply:
Test function name changed, and purpose description added.